### PR TITLE
DOC: update 0.16.0 release notes.

### PR DESCRIPTION
**Pull request by rgommers**
[ci skip]

### CODE REVIEW

File: doc/release/0.16.0-notes.rst [hunk 1]
```diff
@@ -79,6 +98,12 @@ Tukey window was added as `scipy.signal.tukey`.
 The function for computing digital filter group delay was added as
 `scipy.signal.group_delay`.
 
+`scipy.sparse` improvements
+---------------------------
+
+The function `scipy.sparse.norm`, which computes sparse matrix norms, was
+added.
+
```
Comment (ev-br) [inline doc/release/0.16.0-notes.rst]: Maybe also mention `sparse.construct.random`, https://github.com/scipy/scipy/pull/4679, which constructs a sparse matrix with a specified random number distribution.

File: doc/release/0.16.0-notes.rst [hunk 2]
```diff
@@ -94,6 +119,8 @@ The Wishart distribution and its inverse have been added, as
 The Exponentially Modified Normal distribution has been
 added as `scipy.stats.exponnorm`.
```
Comment (ev-br) [inline doc/release/0.16.0-notes.rst]: Maybe also mention `random_state` parameter for stats.distributions, both univariate and multivariate.

File: doc/release/0.16.0-notes.rst (hunk 1 above)
Comment (rgommers) [inline doc/release/0.16.0-notes.rst]: done

File: doc/release/0.16.0-notes.rst (hunk 2 above)
Comment (rgommers) [inline doc/release/0.16.0-notes.rst]: done

File: doc/release/0.16.0-notes.rst [hunk 3]
```diff
@@ -79,6 +98,15 @@ Tukey window was added as `scipy.signal.tukey`.
...
 `scipy.signal.group_delay`.
 
+`scipy.sparse` improvements
+---------------------------
+
+The function `scipy.sparse.norm`, which computes sparse matrix norms, was
+added.
+
+The function `scipy.sparse.random`, which generates sparse matrices with
+normally distributed random samples, was added.
```
Comment (ev-br) [inline doc/release/0.16.0-notes.rst]: Sorry to pester --- "... which allows to draw random variates from an arbitrary distribution". The scope of that PR has changed a bit en route.

Comment (rgommers) [inline doc/release/0.16.0-notes.rst]: Thanks, fixed.

**Comment by rgommers** [comment]
I'll merge this soon if there are no more comments.

**Pull request merged**